src/test_l2_dsVideoDevice.c

```diff
@@ -392,7 +392,7 @@ void test_l2_dsVideoDevice_SetAndVerifyDisplayframerate_sink(void)
             ret = dsGetCurrentDisplayframerate(handle, getframerate);
             UT_ASSERT_EQUAL(ret, dsERR_NONE);
             UT_LOG_DEBUG("dsGetCurrentDisplayframerate return status: %d, framerate: %s", ret, getframerate);
-            UT_ASSERT_STRING_EQUAL(gDSVideoDeviceConfiguration[i].SupportedDisplayFramerate, getframerate);
+            UT_ASSERT_STRING_EQUAL(gDSVideoDeviceConfiguration[i].SupportedDisplayFramerate[j], getframerate);
         } /* for(j) */
     } /* for(i) */
 
```

Comment:
Add one more test case in L2 with 
dsSetFRFMode() & dsGetFRFMode()

Reply:
Added

Reply:
Can you update the 
ds-video-device_L2-Low-Level_TestSpec.md

Reply:
updated